pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -321,11 +321,7 @@ def cmd(self):
             for rsync_dir in Env.rsync_dirs():
                 cmd.append(f'--rsyncdir {rsync_dir}')
             # "--rsyncdir" throws off the configuration root, set it explicitly
-            if 'common/lib' in self.test_id:
-                cmd.append('--rootdir=common/lib')
-                cmd.append('-c common/lib/pytest.ini')
-            elif 'pavelib/paver_tests' in self.test_id:
-                cmd.append('--rootdir=pavelib/paver_tests')
+            cmd.append('--rootdir=pavelib/paver_tests')
```

Comment:
Shouldn't we still be checking for the conditional statement here?
           ` if 'pavelib/paver_tests' in self.test_id:`

Reply:
Previously we had two different test directories so we needed to check the path before adding the `rootdir` value. Since, now only `pavelib/paver_tests` is being tested so we know only the tests from this module will be coming so don't need a condition for the rootdir anymore.